app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/signIn/SignInScreen.kt

```diff
@@ -43,32 +45,64 @@ import com.simply.birthdayapp.presentation.ui.components.AppBaseTopBar
 import com.simply.birthdayapp.presentation.ui.components.AuthButton
 import com.simply.birthdayapp.presentation.ui.components.BaseTextField
 import com.simply.birthdayapp.presentation.ui.components.PasswordTextFiled
-import com.simply.birthdayapp.presentation.ui.extenstions.isPasswordValid
-import com.simply.birthdayapp.presentation.ui.extenstions.isValidEmail
-import com.simply.birthdayapp.presentation.ui.screens.auth.register.RegisterViewModel
 import com.simply.birthdayapp.presentation.ui.theme.AppTheme
 import org.koin.androidx.compose.getViewModel
 
 @Composable
 fun SignInScreen(
-    registerViewModel: RegisterViewModel,
+    signInViewModel: SignInViewModel = getViewModel(),
     onSignInBackClick: () -> Unit = {},
+    onLoginSuccass: () -> Unit = {},
     onForgotPasswordClick: () -> Unit = {}
 ) {
-    val registeredEmail by registerViewModel.registeredEmail.collectAsState()
-    var email by remember { mutableStateOf(registeredEmail) }
-    var password by remember { mutableStateOf("") }
+    val loginSuccess by signInViewModel.loginSuccessState.collectAsState()
+    val loginErrorState by signInViewModel.loginErrorState.collectAsState()
+    val loginErrorMessage by signInViewModel.loginErrorMessage.collectAsState()
+    val hasPasswordError by signInViewModel.hasPasswordError.collectAsState()
+    val hasEmailError by signInViewModel.hasEmailError.collectAsState()
+    val email by signInViewModel.email.collectAsState()
+    val password by signInViewModel.password.collectAsState()
     val checkedState = remember { mutableStateOf(false) }
```

Comment:
Maybe this checkedState can listen to ViewModel and collect it. And when we will call changeRememberPasswordStat, this will also update the checkbox state